crates/cli/src/commands/setup.rs

```diff
@@ -154,38 +136,42 @@ fn already_setup_message(installed_bin_path: PathBuf) {
 }
 
 fn finished_message(
-    installed_bin_path: PathBuf,
-    updated_profile_path: Option<PathBuf>,
+    proto: &ProtoResource,
     exported_content: Option<String>,
+    path_was_updated: bool,
 ) {
-    if let Some(profile_path) = updated_profile_path {
-        println!(
-            "Successfully installed proto to {} and updated profile {}!",
-            color::path(installed_bin_path),
-            color::path(profile_path),
-        );
+    let installed_bin_path = proto.env.store.bin_dir.join(get_exe_file_name("proto"));
+
+    println!(
+        "Successfully installed proto to {}!",
+        color::path(installed_bin_path),
+    );
+
+    if path_was_updated {
         println!("Launch a new terminal window to start using proto!");
+    } else if exported_content.is_some() && cfg!(not(windows)) {
```

Comment:
What about the content & windows case? We can't assume 100% of windows users would want to edit system vars.

Reply:
This is what I was referring to here:

> Note in the finished_message I opted for not suggesting Windows users to configure their PowerShell profile manually, and instead just tell them what to add to the system PATH if they didn't let us do it for them.

Essentially I just think it's far more likely a Windows user will want to update their system PATH as that is effectively what all other programs do. Especially since that's often the only way it'll work with their IDEs.

I can change it back if you'd like.

Reply:
Yeah I'm not sure which is best. I wish there was stats on what users typically do, as I've heard both arguments.

Maybe we show both messages so they can choose?

Reply:
Yeah that sounds preferable. I'll update it to look like this:

```shell
PS D:\Projects\proto> .\target\debug\proto.exe setup --no-modify-path --no-profile
Finishing proto installation...
Successfully installed proto to ~\.proto\bin\proto.exe!
Add the following to your PATH to get started:

C:\Users\caspe\.proto\shims;C:\Users\caspe\.proto\bin

Or alternatively add the following to your shell profile:

# proto
$env:PROTO_HOME = Join-Path $HOME ".proto"
$env:PATH = @(
  (Join-Path $env:PROTO_HOME "shims"),
  (Join-Path $env:PROTO_HOME "bin"),
  $env:PATH
) -join [IO.PATH]::PathSeparator

Need help? Join our Discord https://discord.gg/qCh9MEynv2
```

Unsure about the phrasing, so just shout if you want it changed :)